src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -94,6 +94,13 @@ public function initOrm()
 
         // name for second category
         $categoryTrans2 = new CategoryTranslation();
+        $categoryTrans2->setLocale('en');
+        $categoryTrans2->setTranslation('Second Category');
+        $categoryTrans2->setCategory($category2);
+        $category2->addTranslation($categoryTrans2);
```

Comment:
Postgres and Mysql sort `null` Values at opposite ends of a result set. So in order to properly test the order when sorting after title an existing translation was needed. To test whether or not the API returns blank values for not existing translations I'd suggest creating a new test, which exactly targets that behavior.